ufo/static/userAddForm.html

```diff
@@ -0,0 +1,219 @@
+        <script>
+        var modal = document.getElementById('userModal');
+        var scrollables = modal.getElementsByTagName('paper-dialog-scrollable');
+        for (var i in scrollables) {
+          scrollables[i].dialogElement = modal;
+        }
+        </script>
+      </template>
+    </div>
+
+
+      <!-- <script>
+        var selected_users = [];
+        var checkboxes = document.querySelectorAll('paper-checkbox');
+        var input_elem = document.getElementById('selected_users_array');
+        input_elem.value = JSON.stringify(selected_users);
+
+        for (var i in checkboxes) {
+          var grandParent = checkboxes[i].parentElement.parentElement;
+          var name = grandParent.querySelector('td.directory-user-field.name').innerHTML;
+          var email = grandParent.querySelector('td.directory-user-field.email').innerHTML;
+          checkboxes[i].value = {'name': name, 'email': email};
+
+          checkboxes[i].addEventListener('change', function(event) {
+              var checkboxElem = event.path[0];
+              if (checkboxElem.checked) {
+                selected_users.push(checkboxElem.value);
+              } else {
+                var index = selected_users.indexOf(checkboxElem.value);
+                if (index != -1) {
+                  selected_users.splice(index, 1);
+                }
+              }
+              input_elem.value = JSON.stringify(selected_users);
+          });
+        }
+      </script> -->
+
+      <!--Portion for selecting users to add. -->
+      <!--<form id="users-add-form" method="post" action="{{resources.addUrl}}">
+        <p>Select Users to Add.</p>
+        <table class="padding-between-columns">
+          <tr>
+            <th>-</th>
+            <th>Name</th>
+            <th>Email</th>
+          </tr>
+        {% for directory_user in directory_users %}
+          <tr>
+            <td><paper-checkbox value=""></td>
+            <td class="directory-user-field name">
+              {{ directory_user['name'] }}</td>
+            <td class="directory-user-field email">
+              {{ directory_user['email'] }}</td>
+          </tr>
+        {% endfor %}
+        </table>
+        <input type="hidden" id="selected_users_array" name="users" value="">
+        <input type="hidden" name="_xsrf_token" value="{{ xsrf_token_value }}">
+        <paper-button onclick="submitByFormId('users-add-form')" class="form-submit-button anchor-button" type="submit">Add Selected Users</paper-button>
+      </form>
+
+      <template is="dom-if" if="{{loading}}">
+        <paper-spinner active$={{loading}}></paper-spinner>
+      </template>
+      <iron-ajax
+        url="{{resources.addUrl}}"
+        method="POST"
+        handle-as="json"
+        id="postItemsAjax"
+        loading="{{loading}}"
+        last-response="{{ajaxResponse}}"
+        on-response="bubbleToList">
```

Comment:
nit: can we make the naming of "bubbleToList" more clear?  It's hard to give an example as it's not obvious to me what this is doing, even when I look at the function itself.  :)